bin/p2-inspect/main.go

```diff
@@ -81,7 +81,7 @@ func main() {
 			}
 
 			old := statusMap[podId][node]
-			_, old.Health = health.FindWorst(results)
+			_, old.Health = health.GetMultisourceResult(results)
```

Comment:
We should try to minimize public API changes if possible. The fact that health lives in two places isn't important to clients. `FindWorst` is an important aspect of `pkg/replication` since we always want to ensure we're working on hosts that are the "least healthy" at any given time.